source/strings.tex

```diff
@@ -750,11 +750,9 @@
 For a \tcode{basic_string_view str},
 any operation that invalidates a pointer
 in the range
-\begin{codeblock}
-@\range{str.data()}{\brk{}str.data() + str.size()}@
-\end{codeblock}
+\range{str.data()}{str.data(\linebreak{}) + str.size()}
```

Comment:
Well, this will probably format ugly.
I think you should revert.
The code block was probably added in the first place for something like this.

Reply:
Exactly. I liked it better with the codeblock as well. But as you pointed out, the paper does ask that we inline this, so I relented. Guess @tkoeppe will have to decide what to do here...  (Be careful what you ask for? :) )

Reply:
Removing the code block was suggested by @JohelEGP's original issue submission for [LWG3203](https://wg21.link/lwg3203) :stuck_out_tongue_winking_eye: 

Breaking between the parens does look horrible though:

![image](https://github.com/cplusplus/draft/assets/1254480/23de53cc-0c93-4597-9743-0ba4c539d241)

Keeping the code block seems fine to me. There's no mention of that in the reflector discussion or the Varna minutes, so I don't think LWG requested that. @Dani-Hub, did you add the drafting note to LWG 3203 just because that's what the original proposed resolution did?

Reply:
The original PR [SUPERSERED] only affected [span.overview].

Reply:
Ah yes, so removing it from string view would be for consistency with that. But let's not if it looks bad.

Reply:
Let's retain the codeblock, so as to make this look decent. The paper isn't even quoting its alleged source N4950 correctly, so I don't have any faith in it demanding a particular formatting.